src/Chronos.php

```diff
@@ -998,7 +970,7 @@ public function format(string $format): string
      */
     public function getOffset(): int
     {
-        return $this->native->getOffset();
+        return parent::getOffset();
```

Comment:
Is there much point in retaining methods like these if they only call parent?

Reply:
Only if we want to ensure the type hints are available. I think most IDE's can figure this class out though.

I was thinking about commenting out the ones we don't need to wrap for types, but not delete them. We never know what PHP 9 will bring.